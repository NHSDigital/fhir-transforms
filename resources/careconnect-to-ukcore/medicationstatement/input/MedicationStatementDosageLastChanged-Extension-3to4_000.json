{"resourceType":"MedicationStatement","extension":[{"url":"https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-MedicationDosageLastChanged-1","valueDateTime":"2020-01-28"}]}
