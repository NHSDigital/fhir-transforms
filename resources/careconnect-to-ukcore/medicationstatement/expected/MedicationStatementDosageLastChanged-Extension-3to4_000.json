{
  "resourceType": "MedicationStatement",
  "extension": [
    {
      "extension": [
        {
          "url": "https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-MedicationDosageLastChanged-1",
          "valueDateTime": "2020-01-28"
        }
      ],
      "url": "http://fhir.hl7.org/fhir/3.0/StructureDefinition/extension-Extension-CareConnect-MedicationDosageLastChanged-1"
    }
  ]
}